### PR TITLE
Add 1 icon

### PR DESCRIPTION
**Pull request by SagarDevAchar**
# Description

Added 1 icon

### Icons added
* Filen (`io.filen.app`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

**Comment by x9136** [comment]
Thanks for contributing, the icon fits.

You may want to work on the quality, so I have a few suggestions:

Screenshot 2023-10-17 at 08 05 11
- [The guides](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#foundation) recommend rounding off right angles at 6px. I suggest doing so because it allows icons to have more commonalities and look more appropriate in the library.

- Most icons use a 12px stroke as a base stroke, as described in the guides. Other thicknesses are usually used when 12px degrades the quality of the icon.

- Try to avoid small voids in the circles, because it looks like a superfluous detail in a large size, and especially on a phone screen. [An example](https://github.com/LawnchairLauncher/lawnicons/pull/1414/files#diff-92c069f5c24cddc99c9739ac25171c8a9bf4be7ca5416f83670f5fa90cbe5f8c).

P.S. If you plan to add icons in the future, I suggest you figure out how to update the commit history so that we don't have to scroll through it longer and longer.

**Comment by SagarDevAchar** [comment]
> You may want to work on the quality, so I have a few suggestions:

Updated!

> P.S. If you plan to add icons in the future, I suggest you figure out how to update the commit history so that we don't have to scroll through it longer and longer.

Will fix that before my next contribution.
Apologies for the inconvenience.

File: svgs/filen.svg
Comment (x9136) [inline svgs/filen.svg:1]: Screenshot 2023-10-17 at 10 16 56
As you can see, if you use 12px inside, things get too dense. And that density will increase while the size decreases — on a phone screen, the size is small. On the right is an option if you leave 10px for the center part.

Also you decided to use a fill for some reason. Please use only stroke. In your case it is enough to make 10×10px circles with 10px stroke.

**Review by x9136 — CHANGES_REQUESTED** [review]

**Comment by x9136** [comment]
Perfect 👍 . There's one detail left:

Screenshot 2023-10-17 at 17 35 16

**Pull request merged**